manimlib/scene/scene.py

```diff
@@ -402,9 +403,15 @@ def handle_play_like_call(func):
         def wrapper(self, *args, **kwargs):
             self.update_skipping_status()
             allow_write = not self.skip_animations
-            self.file_writer.begin_animation(allow_write)
-            func(self, *args, **kwargs)
-            self.file_writer.end_animation(allow_write)
+            while True:
+                try:
+                    self.file_writer.begin_animation(allow_write)
+                    func(self, *args, **kwargs)
+                    self.file_writer.end_animation(allow_write)
+                    break
+                except:
```

Comment:
Shouldn't you restrict this `except` temporarily to the error in question? This simply means that all errors will be ignored, which could be damaging depending on the task and the error.